### PR TITLE
[Localization] Capricornus

### PR DESCRIPTION
**Pull request by JavierMatosD**
This PR localizes messages in the following files:
* platform-expressions.cpp
* paragraphs.cpp
* install.cpp
* input.cpp

### CODE REVIEW

File: include/vcpkg/base/messages.h [hunk 1]
```diff
@@ -1220,6 +1232,10 @@ namespace vcpkg
                     "{value} is a localized message name like LocalizedMessageMustNotEndWithNewline",
                     "The message named {value} ends with a newline which should be added by formatting "
                     "rather than by localization.");
+    DECLARE_MESSAGE(ManifestConflict,
+                    (msg::path),
+                    "",
+                    "Found both a manifest and CONTROL files in port {path}; please rename one or the other");
```
Comment (Thomas1664) [inline include/vcpkg/base/messages.h:1238]:
```suggestion
                    "Found both a manifest and CONTROL files in port {path} ; please rename one or the other");
```

Same path issue here.

File: src/vcpkg/paragraphs.cpp [hunk 2]
```diff
@@ -536,9 +536,9 @@ namespace vcpkg::Paragraphs
             {
                 for (auto&& error : results.errors)
                 {
-                    print2(Color::warning, "Warning: an error occurred while parsing '", error->name, "'\n");
+                    msg::println_warning(msgErrorWhileParsing, msg::path = error->name);
                 }
-                print2(Color::warning, "Use '--debug' to get more information about the parse failures.\n\n");
+                msg::println_warning(msgGetParseFailureInfo);
```
Comment (Thomas1664) [inline src/vcpkg/paragraphs.cpp:541]: You're missing one `\n` here.

File: include/vcpkg/base/messages.h [hunk 3]
```diff
@@ -774,9 +779,11 @@ namespace vcpkg
                     (msg::value),
                     "'{value}' is a command line option.",
                     "'--{value}' specified multiple times.");
+    DECLARE_MESSAGE(ElapsedInstallTime, (msg::count), "", "\nTotal elapsed time: {count}");
     DECLARE_MESSAGE(ElapsedTimeForChecks, (msg::elapsed), "", "Time to determine pass/fail: {elapsed}");
     DECLARE_MESSAGE(EmailVcpkgTeam, (msg::url), "", "Send an email to {url} with any feedback.");
     DECLARE_MESSAGE(EmptyArg, (msg::option), "", "The option --{option} must be passed a non-empty argument.");
+    DECLARE_MESSAGE(EmptyInstallPlan, (), "", "Install plan cannot be empty");
```
Comment (BillyONeal) [inline include/vcpkg/base/messages.h:786]: I think this one is a debug assert.

File: include/vcpkg/base/messages.h (hunk 1 above)
Comment (BillyONeal) [reply include/vcpkg/base/messages.h:1238]: This is what the old code did and I don't think the extra space is an improvement; I know I argued against quoting all paths but maybe this is an argument that we should sometimes?

File: include/vcpkg/base/messages.h [hunk 4]
```diff
@@ -1437,6 +1454,7 @@ namespace vcpkg
                     (msg::system_api, msg::exit_code, msg::error_msg),
                     "",
                     "calling {system_api} failed with {exit_code} ({error_msg})");
+    DECLARE_MESSAGE(Test, (), "", "will this compile");
```
Comment (BillyONeal) [inline include/vcpkg/base/messages.h:1457]: :)

File: include/vcpkg/base/messages.h [hunk 5]
```diff
@@ -1493,6 +1511,14 @@ namespace vcpkg
         "{value} is the value provided by the user and {list} a list of unknown variables seperated by comma",
         "invalid argument: url template '{value}' contains unknown variables: {list}");
     DECLARE_MESSAGE(UnrecognizedConfigField, (), "", "configuration contains the following unrecognized fields:");
+    DECLARE_MESSAGE(UnrecognizedIdentifier,
+                    (msg::value),
+                    "'{value}' is an expression identifier",
```
Comment (BillyONeal) [inline include/vcpkg/base/messages.h:1516]:
```suggestion
                    "'{value}' is an expression identifier. For example, given an expression 'windows & x86', 'windows' and 'x86' are identifiers.",
```

**Review by BillyONeal — APPROVED** [review]
LGTM provided the test entry is removed and the typos identified by @Thomas1664 are fixed. (Removing the newlines is OK IMO)

File: include/vcpkg/base/messages.h [hunk 6]
```diff
@@ -865,33 +872,33 @@ namespace vcpkg
...
     DECLARE_MESSAGE(FailedToExtract, (msg::path), "", "Failed to extract \"{path}\":");
+    DECLARE_MESSAGE(FailedToFindPortFeature, (msg::feature, msg::spec), "", "Could not find {feature} in {spec}.");
     DECLARE_MESSAGE(FailedToFormatMissingFile,
                     (),
                     "",
                     "No files to format.\nPlease pass either --all, or the explicit files to format or convert.");
+    DECLARE_MESSAGE(FailedToLoadInstalledManifest,
+                    (msg::spec),
+                    "",
+                    "The control or mnaifest file for {spec} could not be loaded due to the following error. Please "
```
[the remark this answers is not in the document]
Comment (JavierMatosD) [reply include/vcpkg/base/messages.h:883]: Wow, I keep misspelling manifest lol

File: src/vcpkg/paragraphs.cpp (hunk 2 above)
Comment (JavierMatosD) [reply src/vcpkg/paragraphs.cpp:541]: It's ok. I decided to remove it. We aren't consistent about when we add newlines/ how many newlines, and it didn't seem necessary to have two newlines.

File: include/vcpkg/base/messages.h (hunk 3 above)
Comment (JavierMatosD) [reply include/vcpkg/base/messages.h:786]: What do you mean?

**Pull request merged**